packages/flutter_driver/lib/flutter_driver.dart

```diff
@@ -26,6 +26,13 @@ export 'src/common/message.dart' show
   Result;
 export 'src/common/render_tree.dart' show
   RenderTree;
+export 'src/common/wait.dart' show
+  CombinedCondition,
+  FirstFrameRasterized,
+  NoPendingFrame,
+  NoPendingPlatformMessages,
+  NoTransientCallbacks,
+  SerializableWaitCondition;
```

Comment:
Why do you need the `SerializableWaitCondition`?

Reply:
This is the base class that FlutterDriver.waitForCondition exposes. It would be weird to expose just the subclasses and not the base class also. For instance, developers wouldn't be able to make a properly typed collection of wait conditions, or someone writing a helper function that passes the wait condition onto FlutterDriver wouldn't be able to write a properly typed function declaration.